Move utils from benchmarking to syne_tune, and ensure in docs that be… (#606)
awslabs/syne-tune: docs/source/examples.rst

**Requirements**:

> * Needs ``benchmarking`` framework, which requires Syne Tune to be installed

Reviewer: ```suggestion
* Needs Syne Tune's ``benchmarking`` framework, which requires Syne Tune to be installed
```

**Requirements**:

> * Needs ``benchmarking`` framework, which requires Syne Tune to be installed

Reviewer: ```suggestion
* Needs  Syne Tune's ``benchmarking`` framework, which requires Syne Tune to be installed
```

   :caption: examples/launch_pasha_nasbench201.py
   :start-after: # permissions and limitations under the License.

**Requirements**:

> * Needs ``benchmarking`` framework, which requires Syne Tune to be installed
>   `from sources <getting_started.html#installation>`__.

Reviewer: Same suggestion as the other instances:

```suggestion
* Needs  Syne Tune's``benchmarking`` framework, which requires Syne Tune to be installed
  `from source <getting_started.html#installation>`__.
```
